pkg/export/attributes/metric.go

```diff
@@ -111,6 +111,11 @@ var (
 		Prom:    "gpu_memory_allocations_bytes_total",
 		OTEL:    "gpu.memory.allocations",
 	}
+	CapabilityRequests = Name{
+		Section: "capability.requests",
+		Prom:    "capability_requests_total",
+		OTEL:    "capability.requests",
+	}
```

Comment:
I'm not sure how to handle the explosion in cardinality with this metric. There could be lots of PIDs. Maybe we just don't include PID as a label? And instead we could have logs for the more high cardinality information?